bot/exts/utilities/wtf_python.py

```diff
@@ -0,0 +1,122 @@
+{constants.Client.prefix}wtf subclass
+{constants.Client.prefix}wtf del
+```
+If the problem persists send a message in <#{constants.Channels.dev_contrib}>
+"""
+
+MINIMUM_CERTAINTY = 50
+README_REFRESH = 60  # minutes between fetch calls
+
+
+class WTFPython(commands.Cog):
+    """Cog that allows getting WTF Python entries from the WTF Python repository."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.headers: dict[str, str] = {}
+        self.last_fetched = datetime.datetime.now()
+        self.fetch_readme.start()
+
+    @tasks.loop(minutes=README_REFRESH)
+    async def fetch_readme(self) -> None:
+        """Gets the content of README.md from the WTF Python Repository."""
+        refresh = self.last_fetched + datetime.timedelta(minutes=README_REFRESH)
+
+        if refresh > datetime.datetime.now() and self.headers:
+            return  # Cache should be up-to-date
+
+        async with self.bot.http_session.get(f"{WTF_PYTHON_RAW_URL}README.md") as resp:
+            log.trace("Fetching the latest WTF Python README.md")
+            if resp.status == 200:
+                raw = await resp.text()
+                self.parse_readme(raw)
+
+    def parse_readme(self, data: str) -> None:
+        """
+        Parses the README.md into a dict.
+
+        It parses the readme into the `self.headers` dict,
+        where the key is the heading and the value is the
+        link to the heading.
+        """
+        # Match the start of examples, until the end of the table of contents (toc)
+        table_of_contents = re.search(
+            r"\[👀 Examples\]\(#-examples\)\n([\w\W]*)<!-- tocstop -->", data
+        )[0].split("\n")
+
+        for header in list(map(str.strip, table_of_contents)):
+            match = re.search(r"\[▶ (.*)\]\((.*)\)", header)
+            if match:
+                self.headers[match[0]] = f"{BASE_URL}{match[1]}"
+
+    def fuzzy_match_header(self, query: str) -> Optional[str]:
+        """
+        Returns the fuzzy match of a query if its ratio is above "MINIMUM_CERTAINTY" else returns None.
+
+        "MINIMUM_CERTAINTY" is the lowest score at which the fuzzy match will return a result.
+        The certainty returned by rapidfuzz.process.extractOne is a score between 0 and 100,
+        with 100 being a perfect match.
+        """
+        match, certainty, _ = rapidfuzz.process.extractOne(query, self.headers.keys())
+        return match if certainty > MINIMUM_CERTAINTY else None
+
+    @commands.command(aliases=("wtf", "WTF"))
+    async def wtf_python(self, ctx: commands.Context, *, query: str) -> None:
+        """
+        Search WTF Python repository.
+
+        Gets the link of the fuzzy matched query from https://github.com/satwikkansal/wtfpython.
+        Usage:
+            --> .wtf wild imports
+        """
+        match = self.fuzzy_match_header(query)
+        if match:
+            embed = Embed(
+                title=f"WTF Python Search Result For {query}",
```

Comment:
```suggestion
                title=f"WTF Python search result for '{query}'",
```
Additionally, isn't this better added in the description and the title can be something more general?

Reply:
How does this look? 

![image](https://user-images.githubusercontent.com/42116429/135174829-9ed5c78f-ac42-46d2-a5fb-60e9f2384f94.png)

Reply:
I like that, but the capitalization is a bit over-the-top (just do `Search results for 'del'`)